client/src/components/Login/Login.js

```diff
@@ -94,6 +94,7 @@ export default class Login extends React.Component {
                             <Form.Control
                                 type="text"
                                 name="email"
+                                autocomplete="username"
```

Comment:
I didn't know this was a prop. Knarly.

Reply:
https://www.chromium.org/developers/design-documents/form-styles-that-chromium-understands